core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -204,6 +215,12 @@ public void customClasspathResourceMappingTest() throws IOException {
     }
 
     @Test
+    public void customClasspathResourceMappingWithSelinuxTest() throws IOException {
```

Comment:
Sure, we can get the labeled context on the container (provided we use a beefier image with SELinux utils installed), but I thought the :Z option used some host configs to label it as the correct type.  Wouldn't this also require knowledge of some SELinux context on the host to make sure they match?  Thought we'd want to avoid having to interrogate/put dependencies on the host.